Traceback (most recent call last):
  File "test.py", line 293, in <module>
    main()
  File "test.py", line 60, in main
    checkpoints_dir = create_checkpoints_dir(args.dataset, args.fixed, args.mulobj, args.num_blocks, args.hidden_size, args.code_length, args.estimator)
  File "/project/6010010/jing1991/utils.py", line 19, in create_checkpoints_dir
    os.makedirs(dirName)
  File "/home/jing1991/ENV/lib/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: 'checkpoints/fmnist/b1h2048c64/'
